mm2src/coins/tendermint/rpc/ibc.rs

```diff
@@ -0,0 +1,105 @@
+use common::HttpStatusCode;
+use mm2_err_handle::prelude::MmError;
+use mm2_number::BigDecimal;
+
+pub type IBCChainRegistriesResult = Result<IBCChainRegistriesResponse, MmError<IBCChainsRequestError>>;
+pub type IBCTransferChannelsResult = Result<IBCTransferChannelsResponse, MmError<IBCTransferChannelsRequestError>>;
+
+// Global constants for interacting with https://github.com/cosmos/chain-registry repository
```

Comment:
I think relying on our own fork of this repo or a certain commit that is tested first will be better, there are very rare cases where a developer with write access goes rogue and starts adding malicious code/data. What can be the effect of getting a wrong ibc transfer channels? do the coins get burned or transactions just fail?

Reply:
If channel id doesn't exists or it's counterparty channel is not for the token you are sending, tx will fail.

I think we can rely on their commits, we don't need to fork this unless we want to add things into the source.

Reply:
Hmm, if we rely on the commits, we will need to re-build mm2 everytime when they update configurations that effects ibc relays. In this case, forking that repository is better idea, we will only need to fetch upstream changes into fork repository and mm2 will get the updates.